src/MoBi.Presentation/Repositories/MenuBarItemRepository.cs

```diff
@@ -269,6 +270,39 @@ protected override IEnumerable<IMenuBarItem> AllMenuBarItems()
             .WithIcon(ApplicationIcons.Simulation)
             .WithShortcut(Keys.Control | Keys.Alt | Keys.Shift | Keys.S);
 
+         yield return CreateMenuButton.WithCaption(AppConstants.Captions.AddIndividual)
```

Comment:
Were there also buttons removed from this repo?

Reply:
If you remove the buttons, find the relevant id's, tooltips, descriptions etc please.

Reply:
@rwmcintosh I am not removing them yet, since they will be used in the next PR I am currently implementing, about the Ribbon when a module is being selected. If something is left unused after that, I will remove it (although I don't think it is the case).